        On FreeBSD (and maybe others) we need <sys/types.h> for off_t.

diff -ur pmx-2.6.18.orig/libf2c/fio.h pmx-2.6.18/libf2c/fio.h
--- pmx-2.6.18.orig/libf2c/fio.h        2012-05-19 18:47:56.000000000 +0200
+++ pmx-2.6.18/libf2c/fio.h     2012-05-19 18:50:13.501421172 +0200
@@ -1,6 +1,7 @@
 #ifndef SYSDEP_H_INCLUDED
 #include "sysdep1.h"
 #endif
+#include <sys/types.h> /* for off_t */
 #include "stdio.h"
 #ifdef WIN32
 extern int w32getc(FILE *f);

